combination-sum/sungjinwi.py

```diff
@@ -0,0 +1,20 @@
+"""
+    시간 복잡도와 공간복잡도 추후 작성하겠습니다ㅠ
+    풀이 보고 하루 뒤에 기억해서 해보려고 했는데도 한참 걸렸네요
+"""
+class Solution:
+    def combinationSum(self, candidates: list[int], target: int) -> list[list[int]]:
+        ans = []
+        comb = []
+        def recur(n : int):
+            if sum(comb) > target :
+                return
+            elif sum(comb) == target :
+                return ans.append(comb.copy())
```

Comment:
```suggestion
            sum_comb = sum(comb)

            if sum_comb > target :
                return
            elif sum_comb == target :
                return ans.append(comb.copy())
```

sum의 경우, O(n)을 소요하기 때문에 2회 사용하기 보다는 변수에 저장하는 편이 좋을 것 같습니다

Reply:
좋은 리뷰 감사합니다!